### PR TITLE
New article: Delete a microsoft exchange mailbox

### PR DESCRIPTION
**Pull request by DonNateR**
From PR #2105

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2438b9a7c4/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-0ffaaf2de0/how-to/). :bow:

**Comment by kmsholcomb** [comment]
Hi @DonNateR. Edits are attached. I edited both articles to be as similar as possible, so some changes might seem random! If you have a question, see the other article. Thanks! 

[Delete a Microsoft Exchange mailbox_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1146002/Delete.a.Microsoft.Exchange.mailbox_editsKH.pdf)
[Delete a Microsoft Exchange mailbox_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1146001/Delete.a.Microsoft.Exchange.mailbox_editsKH.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-987f0052a4/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8662ad063d/how-to/)

File: content/exchange/delete-a-microsoft-exchange-mailbox.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+4. Select the check box next to the mailbox that you want to delete.
+
+   **Warning:** A deleted mailbox can only be recovered for 14 days. For more information, see [Recover a deleted Microsoft Exchange mailbox](/how-to/recover-a-deleted-microsoft-exchange-mailbox/).
+
+5. Expand the **Select Action** menu at the bottom of the mailbox list, and select **Delete mailboxes**.
+
+   <img src="{% asset_path exchange/delete-a-microsoft-exchange-mailbox/delete-hexbox-sc2.png %}" />
+
+6. Read the information in the popup box that appears, and if there is no conflict, click **Delete 1 mailbox**.
```
Comment (catlook) [inline content/exchange/delete-a-microsoft-exchange-mailbox.md]: ... and, if there is no ...

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9d0320b937/how-to/)

**Pull request merged**